source/jormungandr/jormungandr/interfaces/v1/Places.py

```diff
@@ -175,100 +101,17 @@ def get(self, region=None, lon=None, lat=None):
 
         # If a region or coords are asked, we do the search according
         # to the region, else, we do a word wide search
+
         if any([region, lon, lat]):
-            return self._search_region(args, region, lon, lat)
+            instance = i_manager.get_region(region, lon, lat)
+            response = i_manager.dispatch(args, "places", instance_name=instance)
+            if (not hasattr(response, 'places')) and args['search_type'] == 0:
```

Comment:
no I think this should be in autocomplete.kraken

this concerns only kraken, not Elastic search